docs/DEVELOPMENT.md

```diff
@@ -237,12 +237,17 @@ public function convert_amount($amount, Usage\Resource $resource, $to_unit)
 ```
 
 The `fetch` method returns usage data. For example, you may do a cURL request to an external resource to get usage data. In this example we will always return 100 minutes of ticket time.
+
+The system expects this method to return an array of `UbersmithSDK\Usage\Data` instances. Each Data instance will contain the resource object, the usage amount for the resource as well as an optional `details` parameter.
+
+The content of `details` can be Markdown or html and will be displayed as `Additional details` on invoices as well as in the client portal and client manager.
```

Comment:
The content of `details` can be markdown or html and will be displayed as `Additional details` on invoices as well as in the Client Portal and Client Manager.